i18n/ja/docusaurus-plugin-content-docs/current/user-manual/scripting/esm-scripts.md

```diff
@@ -0,0 +1,116 @@
+---
+title: ESM Scripts
+sidebar_position: 3
+image: /img/user-manual/scripting/esm-script-meta.webp
+---
+
+### A modern way to create PlayCanvas projects
+
+<video width="100%" controls autoPlay loop>
+  <source src="/video/pc-esm-scripts.mp4" type="video/mp4" />
+  Your browser does not support the video tag.
+</video>
+
+ESM Scripts, built around ES Modules, offer a more expressive, flexible, and modern approach to creating interactive PlayCanvas projects. They introduce a modular structure that enhances code organization and maintainability. By allowing you to [import and export code](#importing-modules) across modules, ESM Scripts lead to cleaner, more readable code. Compared to classic scripts, ESM Scripts support static imports, improving performance through more efficient [bundling](#bundling) and dead code-removal.
```

Comment:
Perhaps be explicit that ESM scripts are "more expressive, flexible, etc" over classic scripts.